File: app/src/main/java/me/kirimin/mitsumine/model/Account.java
package me.kirimin.mitsumine.model;

import com.activeandroid.Model;
import com.activeandroid.annotation.Column;
import com.activeandroid.annotation.Table;

@Table(name = "account")
public class Account extends Model {

    @Column(name = "token")
    public String token;

    @Column(name = "tokenSecret")
    public String tokenSecret;

    @Column(name = "url_name")
    public String urlName;

    @Column(name = "display_name")
    public String displayName;

    @Column(name = "profile_image_name")
    public String imageUrl;
}

File: app/src/main/java/me/kirimin/mitsumine/_common/domain/model/Account.kt
package me.kirimin.mitsumine._common.domain.model

import com.activeandroid.Model
import com.activeandroid.annotation.Column
import com.activeandroid.annotation.Table

@Table(name = "account")
open class Account() : Model() {

    constructor(token: String, tokenSecret: String, urlName: String, displayName: String, imageUrl: String) : this() {
        this.token = token
        this.tokenSecret = tokenSecret
        this.urlName = urlName
        this.displayName = displayName
        this.imageUrl = imageUrl
    }

    @Column(name = "token")
    open var token: String = ""

    @Column(name = "tokenSecret")
    open var tokenSecret: String = ""

    @Column(name = "url_name")
    open var urlName: String = ""

    @Column(name = "display_name")
    open var displayName: String = ""

    @Column(name = "profile_image_name")
    open var imageUrl: String = ""
}
